_data-prepper/configuration/data-prepper-reference.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: default
+title: Reference
+has_children: true
+nav_order: 150
+---
+
+This page is a placeholder. Content coming soon.
```

Comment:
I do wonder if there is something else we can put here. Is there a general recommendation for a parent page that would normally not have its own content?

One possible option is to direct readers to either the "data-prepper-configuration" or the "sources/buffers/processors/sinks."

Maybe something like this:

> For general Data Prepper configurations see Data Prepper configuration. For creating a Data Prepper pipeline see Pipeline configuration, Sources, Buffers, Processors, and Sinks.

Reply:
I added this section: 

This page is a placeholder. The Data Prepper reference section provides configuration options to the following Data Prepper components.

- [Data Prepper configuration]({{site.url}}{{site.baseurl}}/opensearch/clients/data-prepper/data-prepper-configuration/)
- [Pipelines]({{site.url}}{{site.baseurl}}/opensearch/clients/data-prepper/pipelines/)
- [Processors]({{site.url}}{{site.baseurl}}/opensearch/clients/data-prepper/processors)
- [Sinks]({{site.url}}{{site.baseurl}}/opensearch/clients/data-prepper/sinks)
- [Sources]({{site.url}}{{site.baseurl}}/opensearch/clients/data-prepper/sources/)
- [Buffers]({{site.url}}{{site.baseurl}}/opensearch/clients/data-prepper/buffers/)

I can still change as needed.